os/board/rtl8730e/src/component/bluetooth/profile/client/rtk_gcs_client.c

```diff
@@ -308,7 +312,12 @@ void general_client_indicate_hdl(void *data)
 	trble_data read_result;
 	
 	read_result.length = indicate_ind->len;
-	read_result.data = malloc(read_result.length);
+	read_result.data = (uint8_t *)malloc(read_result.length);
+	if (!read_result.data)
+	{
+		printf("fail to malloc data \n");
```

Comment:
This is the same as line 293. When you find this log, you can't distinguish where it fails to allocate. Is it ok?

Reply:
I have modified the error log so that it prints the function name.